assets/css/style.css

```diff
@@ -0,0 +1,4 @@
+img{
+    width: 150px;
+    height: 150px;
```

Comment:
to scale down images you should pick either width *or* height but not both. if the original image is 400x200, this scale will cause distortion